src/components/AnonymousChat/MessageChannelItem.tsx

```diff
@@ -72,11 +72,11 @@ const MessageChannelItem: (props: MessageChannelItemProps) => React.ReactElement
       unreadCount={item?.unreadCount}
       isMe={isMe}
       hasFollowButton={followStatus?.isFollowing === false || followStatus?.isFollowingFromAction}
-      // hasFollowButton={true}
       showFollowingButton={followStatus?.isFollowingFromAction}
       handleFollow={onFollowUser}
       channelType={item?.channelType}
       dbAnonUserInfo={dbAnonUserInfo}
+      hasAttachment={item?.rawJson?.message?.attachments?.length > 0}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new `hasAttachment` prop is being used to check if the message has any attachments. This is a good addition as it allows for more granular control over how messages with attachments are handled. However, there might be a potential issue if `item?.rawJson?.message?.attachments` is not an array or undefined. It would be safer to ensure that `attachments` is indeed an array before checking its length.

```diff
- hasAttachment={item?.rawJson?.message?.attachments?.length > 0}
+ hasAttachment={Array.isArray(item?.rawJson?.message?.attachments) && item?.rawJson?.message?.attachments.length > 0}
```
This change ensures that we only check the length of `attachments` if it's an array, preventing potential runtime errors.


<!-- This is an auto-generated comment by OSS CodeRabbit -->